src/components/action-popover/action-popover.spec.js

```diff
@@ -207,6 +214,41 @@ describe("ActionPopover", () => {
     render(null, DOM);
   });
 
+  describe("if download prop and href prop are provided", () => {
+    it("should render as a link component", () => {
```

Comment:
This doesn't actually test that it renders as a link, just that the onClick prop works - you need to look at the elements rendered and assert that it renders as an `<a>`